lib/config/options/index.ts

```diff
@@ -22,17 +22,17 @@ const options: RenovateOptions[] = [
     globalOnly: true,
   },
   {
-    name: 'allowPostUpgradeCommandTemplating',
+    name: 'allowUpgradeCommandTemplating',
```

Comment:
needs migration. simply add them to this list:

https://github.com/renovatebot/renovate/blob/59432b412973a86d99ce7319d93d557473b781db/lib/config/migrations/migrations-service.ts#L75